src/megameklab/com/util/UnitUtil.java

```diff
@@ -2524,7 +2529,6 @@ public static void updateLoadedUnit(Entity unit) {
     }
 
     public static void updateLoadedMech(Mech unit) {
-        UnitUtil.removeOneShotAmmo(unit);
```

Comment:
Do mechs not need one shot ammo removed any more? Or does the removeOneShotAmmo method get called somewhere else more generally?

Reply:
It now gets called in updateLoadedUnit(Entity), so it gets called for all units. This keeps it from showing up in the equipment list.